### PR TITLE
Jlc/fix cdn and css vars

### PR DESCRIPTION
**Pull request by johanseto**
# Description
Fix CDN
This fix the built option to work with CDN.

**Testing Instructions**

Deployed in stage using the caddy gradebook-matcher and know the built works with PUBLIC_PATH using CDN.
## Before

Gradebook MFE with CDN doesn't load.

### Jira story
https://edunext.atlassian.net/browse/FUTUREX-739

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -8,7 +8,7 @@ $fa-font-path: "~font-awesome/fonts";
 @import "~font-awesome/scss/font-awesome";
 
 // $input-focus-box-shadow: $input-box-shadow; // hack to get upgrade to paragon 4.0.0 to work
-$input-focus-box-shadow: var(--pgn-elevation-input-btn-focus-box-shadow);
+$input-focus-box-shadow: var(--pgn-elevation-form-input-base); // cherry-pick style
```
Comment (andrey-canon) [inline src/App.scss:11]: cherry pick from  ?

File: package.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
     "semantic-release": "semantic-release",
     "start": "fedx-scripts webpack-dev-server --progress",
     "test": "TZ=GMT fedx-scripts jest --coverage --passWithNoTests",
-    "watch-tests": "jest --watch"
+    "watch-tests": "jest --watch",
+    "replace-variables": "paragon replace-variables -p src -t usage"
```
Comment (andrey-canon) [inline package.json:20]: I don't understand why you need this ? what is the relation with the cdn ?

File: src/App.scss (hunk 1 above)
Comment (johanseto) [reply src/App.scss:11]: From here lol
https://github.com/eduNEXT/frontend-app-gradebook/pull/3/commits/9ec7b970a7be4fc68e4efc5042bd5246cbb25888#diff-77a5c737b1177a8d6aeae73f7103cb353d22cbb8a944a65dad4fcf3bea142e1eR2

File: package.json (hunk 2 above)
Comment (johanseto) [reply package.json:20]: Actually, I was inspired by this PR.
https://github.com/eduNEXT/frontend-app-gradebook/pull/3
Maybe the CSS support was not necessary. Or what do you think?

Comment (andrey-canon) [reply package.json:20]: This is a definition issue the title is `fix cdn` however the pr that you mention is `add support to css variables and cdn` that makes the difference

**Review by andrey-canon — APPROVED** [review]
LGTM

**Pull request merged**